### DISCUSSION TITLE
File missing

### DISCUSSION THREAD

**Discussion by luchris** [erikdubbelboer/phpRedisAdmin#198] [General]
Hi, I am manually install this app.

First error

PHP Fatal error:  Uncaught Error: Failed opening required '/var/www/html/phpRedisAdmin/includes/../vendor/autoload.php' (include_path='.:/usr/share/pear:/usr/share/php:/usr/share/pear:/usr/share/php') in /var/www/html/phpRedisAdmin/includes/common.inc.php:2
Stack trace:
#0 /var/www/html/phpRedisAdmin/index.php(3): require_once()
#1 {main}

Is this file missing from the package?

Second error

PHP Fatal error:  Uncaught Error: Class "Predis\Client" not found in /var/www/html/phpRedisAdmin/includes/common.inc.php:120
Stack trace:
#0 /var/www/html/phpRedisAdmin/index.php(3): require_once()
#1 {main}
  thrown in /var/www/html/phpRedisAdmin/includes/common.inc.php on line 120
  
What should i do next?

**Comment by erikdubbelboer** [comment]
Please check this section in the readme: https://github.com/erikdubbelboer/phpRedisAdmin#installingconfiguring
